docs/application-tutorial/game_of_life.md

```diff
@@ -0,0 +1,54 @@
+# Game of Life
+
+In the associated [Python file](https://github.com/zama-ai/concrete/blob/main/frontends/concrete-python/examples/game_of_life/game_of_life.py), you can run the Game of Life, written in Concrete Python.
+
+![ Game of Life](game_of_life.gif)
+
+### Installation
+
+In addition to Concrete, you must install `pygame` in your virtual environment:
+
+- `pip3 install pygame`
```

Comment:
added this, removed the requirements.txt

Reply:
Great!